styles/components/dropdown.scss

```diff
@@ -95,4 +59,42 @@
     color: $brand-gray-lt;
     cursor: not-allowed;
   }
+
+  > .menu > a > .icon {
+    padding: ($padding-y / 2);
+  }
```

Comment:
I think we need to put a fixed `width` like `20px` or something (check list-group) here since some icons have varying sizes.